_unittest/conftest.py

```diff
@@ -52,7 +52,7 @@
 from _unittest.launch_desktop_tests import run_desktop_tests
 
 # Initialize default desktop configuration
-default_version = "2022.2"
+default_version = "2023.2"
```

Comment:
you cannot change Unit Test on Build machine since at the moment only 22R2 is installed. You can do it locally on your machine using local_config.json file containing modified default settings 
![image](https://user-images.githubusercontent.com/77293250/214809369-8cd4f418-5cf6-4944-8478-6f96f5ed4d89.png)

Reply:
Sorry, didn't mean to commit that change.